XCTWebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -51,4 +52,11 @@ + (NSArray *)routes
   return FBResponseDictionaryWithOK();
 }
 
++ (id<FBResponsePayload>)handleHideKeyboard:(FBRouteRequest *)request
+{
+    FBXCTSession *session = (FBXCTSession *)[FBSession activeSession];
+    [[session.application.windows elementBoundByIndex:0] tap];
```

Comment:
Could you verify if element you are tapping is expected one? Otherwise it might introduces hard to spot issues